File: test/cs/case21_dict_in.cs
using System.Collections.Generic;
using System.IO;
using System;

public static class Program
{
    public static bool has_key_23(string k)
    {
        Dictionary<string, int> d = new Dictionary<string, int> { { "a", 1 }, { "b", 2 } };
        if (Pytra.CsModule.py_runtime.py_in(k, d))
        {
            return true;
        }
        else
        {
            return false;
        }
    }

    public static void Main(string[] args)
    {
        Pytra.CsModule.py_runtime.print(has_key_23("a"));
    }
}

File: test/rs/case21_dict_in.rs
#[path = "../../src/rs_module/py_runtime.rs"]
mod py_runtime;
use py_runtime::{math_cos, math_exp, math_floor, math_sin, math_sqrt, perf_counter, py_bool, py_grayscale_palette, py_in, py_isalpha, py_isdigit, py_len, py_print, py_save_gif, py_slice, py_write_rgb_png};

// このファイルは自動生成です（native Rust mode）。

fn has_key_23(k: &String) -> bool {
    let mut d: std::collections::HashMap<String, i64> = std::collections::HashMap::from([("a".to_string(), 1), ("b".to_string(), 2)]);
    if py_bool(&(py_in(&(d), k))) {
        return true;
    } else {
        return false;
    }
}

fn main() {
    py_print(has_key_23(&("a".to_string())));
}
